test/unittest/tests/test_reconciliation_tracking.cpp

```diff
@@ -28,6 +28,7 @@ TEST_CASE("Reconciliation tracking: ovfl_track_init", "[reconciliation]")
 
     REQUIRE(__wt_ovfl_track_init(session, &p) == 0);
     REQUIRE(m.ovfl_track != nullptr);
+    __wt_free(session, m.ovfl_track);
```

Comment:
This was to fix a memory leak that was now found because we enabled unittests with the sanitizer builds via the PR. If this should be different, we can pull that out and fix it first. Jeremy made this fix.

Reply:
OK, I'm now tracking this is WT-9548.